### DISCUSSION TITLE
Show calculated checksums in file properties

### DISCUSSION THREAD

**Discussion by Atalanttore** [linuxdeepin/developer-center#3057] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi,

For security-conscious users who nevertheless do not want to do without convenience, it would be good if the properties of a file also displayed its calculated checksums in the common hash methods.

As an example of how to implement something like this, the [HashCheck Shell Extension](http://code.kliu.org/hashcheck/) for Windows comes to mind.

Screenshot:
![image](http://code.kliu.org/hashcheck/screenshots/HashProp2_Aero.png)

Since the mentioned example has not been further developed for more than 10 years, [here](https://github.com/gurnec/HashCheck/) is a forked further developed version.

**Comment by meiyixiang** [comment]
Thank you for your feedback. I have recorded this problem and fed it back to the corresponding product manager. I hope this function can be realized in the future product development.
